test/Npgsql.Tests/NpgsqlParameterTests.cs

```diff
@@ -12,6 +12,8 @@ namespace Npgsql.Tests
     [TestFixture]
     public class NpgsqlParameterTest : TestBase
     {
+        const int ParameterCollectionLookupThreshold = 5;
```

Comment:
Expose an internal const from NpgsqlParameterCollection? We have [InternalsVisibleTo]

Reply:
Ah sure